--- xf86-video-qxl-0.1.4/src/Makefile.am~	2015-04-02 16:25:27.000000000 +0200
+++ xf86-video-qxl-0.1.4/src/Makefile.am	2015-12-29 23:29:46.331193828 +0100
@@ -31,6 +31,7 @@
 	$(SPICE_PROTOCOL_CFLAGS)		\
 	$(XORG_CFLAGS)				\
 	$(PCIACCESS_CFLAGS)			\
+	$(LIBCACARD_CFLAGS)			\
 	$(CWARNFLAGS)				\
 	$(DRM_CFLAGS)				\
 	$(LIBUDEV_CFLAGS)
--- xf86-video-qxl-0.1.4/src/spiceccid/Makefile.am~	2015-03-30 16:58:56.000000000 +0200
+++ xf86-video-qxl-0.1.4/src/spiceccid/Makefile.am	2015-12-29 23:31:25.765454723 +0100
@@ -19,7 +19,7 @@
 #  CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 
-AM_CFLAGS = $(LIBPCSCLITE_CFLAGS)
+AM_CFLAGS = $(LIBPCSCLITE_CFLAGS) $(LIBCACARD_CFLAGS)
 
 if BUILD_SPICECCID
 libspiceccid_la_LTLIBRARIES = libspiceccid.la
